scripts/v.dissolve/v.dissolve.html

```diff
@@ -1,126 +1,176 @@
 <h2>DESCRIPTION</h2>
 
-The <em>v.dissolve</em> module is used to merge adjacent areas in a vector
-map which share the same category value. The resulting merged area(s) retains
-this category value. Alternatively an integer or string column can be defined
-which is used to find adjacent polygons with identical attribute for common
-boundary dissolving. In this case the categories are not retained, only the
-values of the new key column. See the <em>v.reclass</em> help page for details.
+The <em>v.dissolve</em> module is used to merge adjacent or overlapping 
+features in a vector map that share the same category value. The 
+resulting merged feature(s) retain this category value. 
 
 <center>
 <img src="v_dissolve_zipcodes.png">
 <img src="v_dissolve_towns.png">
 <p><em>
-    Figure: Areas with the same attribute value (first image) are
-    merged into into one (second image)
+Figure: Areas with the same attribute value (first image) are merged 
+into one (second image).
 </em></p>
 </center>
 
-<h3>Merging behavior</h3>
+<p>
+
+Instead of dissolving features based on the category values, the user 
+can define an integer or string column using the <b>column</b> 
+parameter. In that case, features that share the same value in that 
+column are dissolved. Note, the newly created layer does not retain the 
+cat values from the input layer, only the values of the new key column.
 
-Multiple areas with the same category or the same attribute value
-which are not adjacent are merged together into one entity
-which consists of multiple areas, i.e., a multipolygon.
+<p>
+Note that multiple areas with the same category or the same attribute 
+value that are not adjacent are merged into one entity, which consists 
+of multiple features. I.e., this results in multipart features.
```

Comment:
```suggestion
Note that multiple areas with the same category or the same attribute 
value that are not adjacent are merged into one entity, which consists 
of multiple features, i.e., a multipart feature.
```

Or simply start a new sentence and restate the same in other words (Or say "In other words, ...").